lib/aggregate-translations.js

```diff
@@ -45,6 +45,11 @@ const defaults = (options = {}) => {
   return defaultConfig;
 };
 
+/**
+ * Aggregates translation files and initializes the appropriate internationalization loaders.
+ * @param {Object} options - The aggregation options.
```

Comment:
If we are adding inline-docs, can we list all of the supported options?

Reply:
Sure thing, I had these listed originally but they are also defined in the root README page so I decided to remove them. I'll update the docs for this function.

Reply:
Added descriptions here: https://github.com/cerner/terra-aggregate-translations/pull/19/commits/5d41f0d4a7c507494596f40e4584e87d8223e25b

For those curious about the format I tried to match the recommendations from JS Docs.

https://jsdoc.app/tags-param.html#parameters-with-properties

https://jsdoc.app/tags-param.html